src/main/java/de/tum/in/www1/artemis/service/user/UserService.java

```diff
@@ -338,25 +341,58 @@ private User handleRegisterUserWithSameLoginAsExistingUser(User newUser, User ex
         throw new AccountRegistrationBlockedException(existingUser.getEmail());
     }
 
+    // TODO use this code
```

Comment:
unresolved TODO

Reply:
The TODO has been resolved before, but now it also got removed